examples/vue-kitchen-sink/src/stories/addon-controls.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta, Preview, Story } from '@storybook/addon-docs/blocks';
+import MyButton from './Button.vue';
+
+<Meta
+  title="Addon/ControlsMDX"
+  component={MyButton}
+  argTypes={{
+    color: { control: { type: 'color' } },
+  }}
+/>
+
+export const ButtonStory = (args) => ({
+  props: Object.keys(args),
+  components: { MyButton },
+  template: '<my-button :color="color" :rounded="rounded">{{label}}</my-button>',
+});
+
+# Addon-controls in MDX
+
+Controls can also be defined and used in MDX stories.
+
+## Rounded
+
+<Preview>
+  <Story
+    name="Rounded"
```

Comment:
Just wanted to say this is super helpful, @shilman, as someone who's been trying out the beta versions in our large-ish Vue storybook, & trying to get args + .mdx working. Thank you!

Reply:
yup a big thanks from me too, trying to get examples of different plugins working MDX and vue has been a real pain. As knobs stopped working in v6 I moved over to controls, this is a great example and made my life way easier, thanks @shilman

Reply:
Why did knobs stop working? Is there an open issue i can look at? Thanks!

Reply:
I'm not sure at what point it stopped working (I'm sure it worked at some point in v6, I can always checkout old versions and test as I tend to update each week). Knobs still worked for standard Web elements, but not as props for vue components.

Components started complaining that all props were strings. While debugging I found that even for components with just string props the components weren't being updated when updating the knob and the default values from the component were being used over the default knob values.

I wasn't sure if it was something to do with my configuration, or if knobs had dropped support for vue for v6 in favour of this addon so I didn't raise an issue. 

If I get time tomorrow (otherwise Monday) I'll see at what point it broke and let you know. It should be really simple to create an example from the kitchen sink to test it